.circleci/config.yml

```diff
@@ -563,6 +563,11 @@ jobs:
       - store_test_results:
           path: ~/react-native/reports/junit
 
+      - run:
```

Comment:
Can you move this step to just before # Collect Results?

Reply:
@hramos done.